apps/web/src/components/dashboard/energy-consumption-tooltip.tsx

```diff
@@ -54,14 +54,25 @@ export default function EnergyConsumptionTooltip({ payload }: TooltipProps<Value
         }
     };
 
+    const computeTimestampLabel = (aggregationParam) => {
+        switch (aggregationParam) {
+            case AggregationType.YEAR: return ' Wh / Jahr';
```

Comment:
In den Zeilen 59-64 bitte jeweils das Leerzeichen am Anfang entfernen. In Z. 75 hast du nämlich zwischen `{energy}` und `{computeTimestampLabel(aggregation)}` ein Leerzeichen stehen. Damit hast du dann insgesamt jeweils zwei Leerzeichen